internal/operators/ocs/validations.go

```diff
@@ -120,19 +120,23 @@ func (o *operator) computeNodeResourceUtil(host *models.Host, nodeInfo *resource
 		return status, err
 	}
 
-	disks := getValidDiskCount(inventory.Disks)
-	if disks > 1 { // OCS must use only the non-boot disks
-		requiredDiskCPU := int64(disks-1) * o.config.OCSRequiredDiskCPUCount
-		requiredDiskRAM := int64(disks-1) * o.config.OCSRequiredDiskRAMGiB
+	disks, err := getValidDiskCount(inventory.Disks)
+	if err != nil {
+		return err.Error(), err
+	}
+
+	if disks > 0 { // OCS must use only the non-boot disks
+		requiredDiskCPU := disks * o.config.OCSRequiredDiskCPUCount
+		requiredDiskRAM := disks * o.config.OCSRequiredDiskRAMGiB
 
 		if inventory.CPU.Count < requiredDiskCPU || inventory.Memory.UsableBytes < conversions.GibToBytes(requiredDiskRAM) {
-			status = fmt.Sprint("Insufficient resources on host with host ID ", *host.ID, " to deploy OCS. The hosts has ", disks, " disks that require ", requiredDiskCPU, " CPUs, ", requiredDiskRAM, " GiB RAM")
+			status = fmt.Sprint("Insufficient resources on host with host ID ", *host.ID, " to deploy OCS. The hosts has ", disks, " non-bootable disks that require ", requiredDiskCPU, " CPUs, ", requiredDiskRAM, " GiB RAM")
```

Comment:
Maybe we can make this status message more specific and inform the user what's missing (not what's required)

Reply:
I was thinking about changing all the Validation messages in a different PR.